next.config.js

```diff
@@ -1,9 +1,4 @@
-const withPlugins = require('next-compose-plugins')
-
-module.exports = withPlugins([], {
-  webpack: (config) => {
```

Comment:
Great! I think this was only there because of linting that used to run in the Webpack pipeline but was then moved outside of the build step (58a73d4ab0e655d6cc019aecf052b2b1ca227132).

Reply:
makes sense! hell yea. Let's go!